app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,7 @@
     <application
         android:name=".ProjmagApplication"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@mipmap/ic_projmag_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
```

Comment:
On my emulator, it doesn't work unless I also change the roundIcon to the new icon.
```suggestion
        android:icon="@mipmap/ic_projmag_launcher"
        android:label="@string/app_name"
        android:roundIcon="@mipmap/ic_projmag_launcher"
```

Reply:
Thanks, I forgot it. It is fixed now